packages/blocks/package.json

```diff
@@ -63,9 +63,9 @@
     "remark-parse": "^11.0.0",
     "remark-stringify": "^11.0.0",
     "shiki": "^1.14.1",
+    "simple-xml-to-json": "^1.2.2",
```

Comment:
I personally recommend https://lezer.codemirror.net/

Reply:
https://github.com/lezer-parser/xml

Reply:
It may take some time to adopt this module. I'll come back later.